crates/neon-macros/Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["Dave Herman <david.herman@gmail.com>"]
 description = "Procedural macros supporting Neon"
 repository = "https://github.com/neon-bindings/neon"
 license = "MIT/Apache-2.0"
-edition = "2018"
+edition = "2021"
```

Comment:
Please revert this change until we can discuss bumping the minimum supported rust version.